target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch

```diff
@@ -0,0 +1,13 @@
+--- a/drivers/mtd/spi-nor/spi-nor.c
```

Comment:
Add your commit description also to the top of the patch file. This way, the intention of the patch is immediately clear to the reader.

Reply:
I added my commit. Different patches do that in different ways. I hope the way I decided is okay? :)